src/main/java/org/opensearch/security/dlic/rest/validation/SavedTenantValidator.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Portions Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package org.opensearch.security.dlic.rest.validation;
+
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.rest.RestRequest;
+
+/**
+ * Validator for Saved Tenant API Action.
+ */
+public class SavedTenantValidator extends CredentialsValidator {
+    public SavedTenantValidator(RestRequest request, BytesReference ref, Settings opensearchSettings, Object... param) {
+        super(request, ref, opensearchSettings, param);
+        allowedKeys.put("saved_tenant", DataType.STRING);
+        mandatoryKeys.add("saved_tenant");
+        /*
```

Comment:
Why is this comment needed here?

Reply:
It isn't; it was a note for myself for another feature 😱 . Removed